.eslintrc.json

```diff
@@ -3,7 +3,7 @@
   "globals": {
     "Prism": true
   },
-  "plugins": ["react"],
+  "plugins": ["react", "@typescript-eslint"],
```

Comment:
For my own understanding, what is this `react` plugin for? 🤔

Reply:
It is used for linting .mdx files. The react plugin fixes ESLint's JSX support, by marking variables as "used", if I remember right. I think that without it something like this would fail:

```js
import {Component} from 'package'
// It would say that component is imported but never used

<Component />
```